src/Language/PureScript/TypeChecker.hs

```diff
@@ -490,7 +575,7 @@ typeCheckModule (Module ss coms mn decls (Just exps)) =
   -- Check that all the classes defined in the current module that appear in member types have also
   -- been exported from the module
   checkClassesAreExported :: DeclarationRef -> m ()
-  checkClassesAreExported ref = checkMemberExport findClasses ref
+  checkClassesAreExported ref = checkMemberExport findClasses  ref
```

Comment:
Sorry to nitpick but would you mind reverting this? I'm just trying to keep `git blame` useful where we can.

Reply:
Not a problem at all, I should have caught it :)